common/mount.py

```diff
@@ -14,6 +14,9 @@
 #    with this program; if not, write to the Free Software Foundation, Inc.,
 #    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 
+"""Modul docstring
+"""
```

Comment:
"""High-level mount API for mounting, umounting, checks etc.
"""

Reply:
It is a comment not a file modification. Not sure how the system reacts when I commit on top of your suggested changes. ;)

I will keep that in mind and modify it later, OK?

Reply:
I edited that file directly in GitHub and added the commit to my branch. Seems like it worked.